app/views/templates/project-recent-activity.html.slim

```diff
@@ -0,0 +1,35 @@
+.recent-activity
+  p.loading ng-if='!reports'
+    em Loading...
+  p.empty ng-if='reports.length === 0'
+    em No tests have been run recently.
+  .list-group ng-if='reports.length'
+    .list-group-item ng-repeat='report in reports'
+      span.singleRunner ng-if='report.runners.length == 1'
+        user-avatar ng-if='report.runners.length == 1' user='report.runners[0]'
+        strong.name {{ report.runners[0].name }}
+      span.multipleRunners ng-if='report.runners.length >= 2'
+        span.runner ng-repeat='runner in report.runners'
+          user-avatar user='runner' name-tooltip='true'
+      '  ran some tests
```

Comment:
I use `'` in place of `|` because of trailing space removed by my IDE. In fact, `'` add automatically a trailing space.